### PR TITLE
load issue evidence tab content with ajax

### PR DESCRIPTION
**Pull request by MattBudz**
### Summary

Clean up and improve the performance of `issues#show` by loading the evidence tab content with ajax

### Check List

- [x] Added a CHANGELOG entry

### CODE REVIEW

File: app/controllers/issues/evidence_controller.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+  def all_evidence_columns
+    @issue.evidence
+          .map { |evidence| evidence.fields.keys }
+          .flatten
+          .uniq - ['Title', 'Label']
+  end
+
+  def set_issues
+    @issues = Issue.where(node_id: current_project.issue_library.id)
```
Comment (etdsoft) [inline app/controllers/issues/evidence_controller.rb:21]: Why do you need to load this collection?

File: app/views/issues/_evidence.html.erb [hunk 2]
```diff
@@ -1,24 +1,8 @@
...
-  </div>
-<% end %>
+<%= content_tag :div,
+  spinner_tag,
+  data: {
+    behavior: 'fetch',
+    path: project_issue_evidence_index_path(issue_id: @issue.id)
+  }
+%>
+
```
Comment (etdsoft) [inline app/views/issues/_evidence.html.erb:8]: Do we need a partial any more? It seems that this chunk can go into show.html.erb 

Aesthetically, once there, I think I'd move the `content_tag` to a new line nested under `<%=`

File: app/views/issues/evidence/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,23 @@
+<h4 class="header-underline">Evidence information</h4>
+<% if @issue.affected.any? %>
+  <% cache ['issue-evidence-tab', @issue, @issue.evidence] do %>
+    <%= render 'issues/evidence/table' %>
+  <% end %>
+<% else %>
```
Comment (etdsoft) [inline app/views/issues/evidence/index.html.erb:6]: this is inconsistent (I believe) with Nodes#show>Evidence:empty

https://github.com/dradis/dradis-ce/blob/develop/app/views/nodes/show/_evidence.html.erb

File: app/views/layouts/tylium.html.erb [hunk 4]
```diff
@@ -82,7 +82,7 @@
       <%= render 'boards/modals/new_board_form' unless request.xhr? %>
     <% end %>
 
-    <% if controller_name == 'evidence'%>
+    <% if controller_path == 'evidence' %>
```
Comment (etdsoft) [inline app/views/layouts/tylium.html.erb:85]: 👏

File: app/controllers/issues/evidence_controller.rb (hunk 1 above)
Comment (MattBudz) [reply app/controllers/issues/evidence_controller.rb:21]: good question, removed.

File: app/views/issues/evidence/index.html.erb (hunk 3 above)
Comment (MattBudz) [reply app/views/issues/evidence/index.html.erb:6]: updated

**Pull request merged**